src/answers-umd.js

```diff
@@ -163,6 +164,12 @@ class Answers {
       resetListener: data => {
         if (!data[StorageKeys.QUERY]) {
           this.core.clearResults();
+        } else {
+          this.core.globalStorage.set(StorageKeys.QUERY_TRIGGER, QueryTriggers.QUERY_PARAMETER);
+        }
+
+        if (!data[StorageKeys.SEARCH_OFFSET]) {
```

Comment:
See note in the commit message. I made SLAP-616 to address going from presence to absence of query param on back nav.